toolkit/partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs

```diff
@@ -17,6 +20,15 @@ pub fn prepare_main_chain_config<C: IOContext>(
 	let cardano_parameteres = prepare_cardano_params(ogmios_config, context)?;
 	cardano_parameteres.save(context);
 	set_up_cardano_addresses(context, genesis_utxo, ogmios_config)?;
+
+	let offchain = context.offchain_impl(ogmios_config)?;
+	let (payment_key, governance_authority) = get_private_key_and_key_hash(context)?;
+
+	let runtime = tokio::runtime::Runtime::new().map_err(|e| anyhow::anyhow!(e))?;
+	runtime
+		.block_on(offchain.init_governance(governance_authority, payment_key, genesis_utxo))
+		.map_err(|e| anyhow::anyhow!("Offchain call failed: {e:?}!"))?;
```

Comment:
I try to think like a user. I see: "Offchain call failed: <something>" and now I'm wondering what an "Offchain call" is, but I think in v1.4 we will still have "beta" experience of some CLIs.